source/language/openpulse.rst

```diff
@@ -238,7 +238,7 @@ Waveforms
 
 Waveforms are of type ``waveform`` and can either be:
 
-- An array of complex samples (note this syntax is still under [active development](https://github.com/Qiskit/openqasm/pull/301) and is subject to change.) which define the points for the waveform envelope
+- An array of complex samples which define the points for the waveform envelope
```

Comment:
I don't feel confident enough on the openpulse development to approve this change, but I suspect it's fine.

Reply:
The pull request referenced is closed, so the text should change somehow, but happy to change it another way if someone has a different suggestion.

Reply:
For what it's worth, in our implementation of OpenPulse we've moved away from this style of waveform declaration. I wrote up a few more details here https://github.com/openqasm/openqasm/pull/414#issuecomment-1839633685.